alpha/action/testdata/index-declcfgs/exp-latest/index.yaml

```diff
@@ -43,35 +49,14 @@ defaultChannel: stable
 name: baz
 schema: olm.package
 ---
-image: test.registry/baz-operator/baz-bundle:v1.0.1
```

Comment:
Hmm. Need to investigate why this is removed. Perhaps its because the only reason is showed up in the expected output is because of differences in the channel-related metadata?

Reply:
Yep, just confirmed that `image: test.registry/baz-operator/baz-bundle:v1.0.1` is now identical in both `old/index.yaml` and `latest/index.yaml`.

On `master`, this bundle has an extra `olm.skips` property for `baz.v1.0.0` in `old/index.yaml` that is not present in `latest/index.yaml`

Since this skips field was moved to the `olm.channel` in this PR, the bundles are now identical and the diff correctly omits it from the result.